symbolic-il2cpp/src/usym.rs

```diff
@@ -26,16 +120,16 @@ mod raw {
         /// These follow right after the header, and after them is the string table.
         pub(super) record_count: u32,
 
-        /// UUID of the assembly, offset into string table.
+        /// UUID of the assembly, as an offset into string table.
```

Comment:
```suggestion
        /// UUID of the assembly, as an offset into the string table.
```

Reply:
since you're changing all these you might also want to take into account that we're not longer calling this a "string table" for usym, it's `UsymSymbols::strings` now.

Reply:
i've changed the phrasing a little everywhere in light of @flub's comment. i'm calling it a strings section now (🎻) which i'm hoping is an acceptable substitute for the time being, but i'm open to different suggestions ~~so long as they're not related to tables~~ if there are any issues with the new name.

Reply:
🎻 ❤️